TST: flake8 should run on all python builds

From PR #39 , 
I received the following warning when running `flake8` on branch `tswast:gbq-37` but Travis did not report the warning. 

Local run:
```
tony@tonypc:~/pydata-pandas-gbq$ flake8 pandas_gbq 
pandas_gbq/tests/test_gbq.py:136:61: F841 local variable 'e' is assigned to but never used
```

On Travis:
https://travis-ci.org/tswast/pandas-gbq/jobs/234914390

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,7 +3,7 @@ sudo: false
 language: python
 
 env:
-  - PYTHON=2.7 PANDAS=0.19.2 COVERAGE='false' LINT='false'
+  - PYTHON=2.7 PANDAS=0.19.2 COVERAGE='false' LINT='true'
   - PYTHON=3.5 PANDAS=0.18.1 COVERAGE='true' LINT='false'
   - PYTHON=3.6 PANDAS=0.20.1 COVERAGE='false' LINT='true'
   - PYTHON=3.6 PANDAS=MASTER COVERAGE='false' LINT='false'

